adagrams/game.py

```diff
@@ -1,11 +1,95 @@
+import random
+import copy
+
+LETTER_POOL = {
```

Comment:
Good data structure choice and also good to keep it in the main module scope like this.